webtau-utils/src/main/java/org/testingisdocumenting/webtau/utils/JsonUtils.java

```diff
@@ -129,4 +129,8 @@ public static Object deserialize(String json) {
     public static JsonNode convertToTree(Object object) {
         return mapper.valueToTree(object);
     }
+
+    public static <T> T convert(Object fromValue, Class<T> toValueType) {
```

Comment:
do we need a test for this?

Reply:
We could add one but given it's an incredibly straightforward passthrough I'm inclined not to.